doc/source/learning/airship.rst

```diff
@@ -0,0 +1,175 @@
+Airship
+=====================
+
+Airship is a platform that takes YAML documents and turns them into Clouds.
+Takes helm charts, baremetal, security policies and network description and creates a cloud
+
+.. image:: https://airship-treasuremap.readthedocs.io/en/latest/_images/architecture.png
```

Comment:
I would replace latest by 1.0 so that this is stable.

Reply:
Fixed